samples/nrf9160/mqtt_simple/prj.conf

```diff
@@ -0,0 +1,38 @@
+# General config
+CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
We don't have entropy driver for nRF9160 yet? Just curious.

Reply:
No :)